packages/react-reconciler/src/ReactFiberCacheComponent.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {ReactContext} from 'shared/ReactTypes';
+
+import {REACT_CONTEXT_TYPE} from 'shared/ReactSymbols';
+
+export type Cache = {|
+  providers: Set<Fiber> | null,
+  data: Map<() => mixed, mixed> | null,
+|};
+
+export const CacheContext: ReactContext<Cache | null> = {
+  $$typeof: REACT_CONTEXT_TYPE,
+  // We don't use Consumer/Provider for Cache components. So we'll cheat.
+  Consumer: (null: any),
+  Provider: (null: any),
```

Comment:
Will that affect JIT level optimizations you think?

Reply:
Wasn't sure, figured you would tell me :D